src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -0,0 +1,70 @@
+package com.kodilla.ecommercee.domain;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "products")
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Product {
+
+    @Column(name = "id")
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(name = "name")
+    @NotNull
+    private String name;
+
+    @Column(name = "price")
+    @NotNull
+    private double price;
```

Comment:
Never use double type for variables that store constant precision here is why: https://wiki.sei.cmu.edu/confluence/display/java/NUM04-J.+Do+not+use+floating-point+numbers+if+precise+computation+is+required
```suggestion
    private BigDecimal price;
```